src/components/navbar.js

```diff
@@ -40,38 +42,14 @@ const Navbar = () => {
 		}
   `)
 
-  React.useEffect(() => {
-		// Get all "navbar-burger" elements
-		const $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
-	
-		// Check if there are any navbar burgers
-		if ($navbarBurgers.length > 0) {
-	
-			// Add a click event on each of them
-			$navbarBurgers.forEach( el => {
-				el.addEventListener('click', () => {
-	
-					// Get the target from the "data-target" attribute
-					const target = el.dataset.target;
-					const $target = document.getElementById(target);
-	
-					// Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
-					el.classList.toggle('is-active');
-					$target.classList.toggle('is-active');
-	
-				})
-			})
-
-			return function cleanup() {
-				$navbarBurgers.forEach(el => el.removeEventListener('click'))
-			}
-		}
-	})
-
 	return (
 		<nav className="navbar is-light is-fixed-top is-spaced container" role="navigation" aria-label="main navigation">
 			<div className="navbar-brand">
-				<a role="button" className="navbar-burger burger is-marginless" aria-label="menu" aria-expanded="false" data-target="navbarMenu">
+				<a role="button" className={`navbar-burger burger is-marginless ${ isBurgerActive ? 'is-active' : ''}`} aria-label="menu" aria-expanded="false"
+					onClick={() => {
+						setIsBurgerActive(!isBurgerActive);
+						document.getElementById("navbarMenu").classList.toggle('is-active');
```

Comment:
This line isn't really the react way.
I was tempted to do the class thing directly on the navBarMenu (same as on the a tag) but this links the logic together nicely.

Reply:
Yeah I see what you're saying. I think it is fine for now.